subsys/bluetooth/controller/ll_sw/nordic/lll/lll_conn.c

```diff
@@ -174,7 +175,8 @@ void lll_conn_isr_rx(void *param)
 	if (crc_ok) {
 		uint32_t err;
 
-		err = isr_rx_pdu(lll, pdu_data_rx, &tx_release, &is_rx_enqueue);
+		err = isr_rx_pdu(lll, pdu_data_rx, &is_rx_enqueue, &tx_release,
+				 &is_done);
```

Comment:
is_done gets a value in isr_rx_pdu, but this value is not used. In line 205 there is a new assignment to is_done. Am I missing something here?

Reply:
Good catch, I messed up a conflict resolution, is_crc_backoff needs to be renamed to is_done. Will fix it.